build-support/migration-support/migrate_to_toml_config_test.py

```diff
@@ -0,0 +1,228 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pathlib import Path
+from textwrap import dedent
+
+from migrate_to_toml_config import generate_new_config
+
+from pants.util.contextutil import temporary_dir
+
+
+def assert_rewrite(*, original: str, expected: str) -> None:
+  with temporary_dir() as tmpdir:
+    build = Path(tmpdir, "pants.ini")
+    build.write_text(original)
+    result = generate_new_config(build)
+  assert result == expected.splitlines()
+
+
+def test_fully_automatable_config() -> None:
```

Comment:
Thank you for writing great tests for this script and acknowledging that everything we ship to users should be tested!!! This makes me happy.